src/nvim/auevents.lua

```diff
@@ -29,6 +29,7 @@ return {
     'CmdWinLeave',            -- before leaving the cmdline window
     'ColorScheme',            -- after loading a colorscheme
     'ColorSchemePre',         -- before loading a colorscheme
+    'CompleteChange',         -- after change completion item
```

Comment:
probably should be named `CompleteChanged` (similar to `CmdLineChanged`, `TextChanged`, etc.)

Reply:
Fixed